src/Microsoft.SqlTools.Serialization/Serialization.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.SqlTools.Hosting.Contracts;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.Serialization
+{
+    public class SaveResultsInfo
```

Comment:
Isn't this a contract? You should have this in a `Contracts` folder with a `Microsoft.SqlTools.Serialization.Contracts` namespace. Also, I'd recommend putting this alongside all the other contracts, e.g. those used in query execution. You're likely going to need this for your handlers which can all be in the ServiceLayer codebase. All this executable DLL should be for is registering the service, everything else can be in a core location.